lib/iris/analysis/__init__.py

```diff
@@ -2407,35 +2405,26 @@ def regridder(self, src_cube, target_grid):
         """
         Creates a nearest-neighbour regridder to perform regridding from the
         source grid to the target grid.
-
         This can then be applied to any source data with the same structure as
         the original 'src_cube'.
-
         Typically you should use :meth:`iris.cube.Cube.regrid` for
         regridding a cube. There are, however, some situations when
         constructing your own regridder is preferable. These are detailed in
         the :ref:`user guide <caching_a_regridder>`.
-
         Args:
-
         * src_cube:
             The :class:`~iris.cube.Cube` defining the source grid.
             The X and Y coordinates must be mapped over the same dimensions.
-
         * target_grid:
             The :class:`~iris.cube.Cube` defining the target grid.
             It must have only 2 dimensions.
             The X and Y coordinates must be one-dimensional and mapped to
             different dimensions.
-
         Returns:
             A callable with the interface:
-
                 `callable(cube)`
-
             where `cube` is a cube with the same grid as `src_cube`
             that is to be regridded to the `target_grid`.
-
         """
         from iris.analysis.trajectory import \
             UnstructuredNearestNeigbourRegridder
```

Comment:
the changes left in this file are non-functional, as far as I can see.  If so, please may we revert them, to reduce the change noise?